src/scripts/main.js

```diff
@@ -1,3 +1,422 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+
+let sortedRows = [];
+
+table.tHead.addEventListener('click', (e) => {
+  sortEmployees(e.target.textContent, e.target);
+});
+
+selectRow();
+
+createMsgBlock();
+
+function getIdx(title) {
+  const idx = 0;
+
+  switch (title) {
+    case 'Position':
+      return 1;
+    case 'Office':
+      return 2;
+    case 'Age':
+      return 3;
+    case 'Salary':
+      return 4;
+    default:
+      return idx;
+  }
+}
+
+function sortEmployees(title, elem) {
+  const idx = getIdx(title);
+
+  switch (title) {
+    case 'Age':
+    case 'Salary':
+      if (elem.className === '' || elem.className === 'DESC') {
+        sortedRows = [...table.tBodies[0].children].sort((a, b) => {
+          if (
+            +[...a.children][idx].textContent.replace(/[^0-9.]/g, '') >
+            +[...b.children][idx].textContent.replace(/[^0-9.]/g, '')
+          ) {
+            return 1;
+          }
+
+          if (
+            +[...a.children][idx].textContent.replace(/[^0-9.]/g, '') <
```

Comment:
you can create a function to normalize it instead of repeating code a few times

Reply:
normalize function is created